.github/scripts/run-v0-devbig.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# This script runs TorchBench v0 without installing the package
+# It assumes pytorch and its dependencies have been installed correctly
+# Usage:
+# run-v0-devbig.sh RESULT_DIR BENCHMARK_FILTER CONDA_ENV_NAME
+# All three arguments are required
+
+set -eo pipefail
+
+# Big machine config
+DATA_DIR=$1
+BENCHMARK_FILTER="$2"
+CONDA_ENV_NAME=$3
+CURRENT_DIR=$(dirname "$(readlink -f "$0")")
+
+# Install benchmark dependencies
+python install.py
+
+sudo -E systemd-run --slice=workload.slice --same-dir --wait --collect --service-type=exec --pty --uid=$USER \
```

Comment:
what is systemd-run helping with here? not familiar with it..

Reply:
> what is systemd-run helping with here? not familiar with it..

This is to coordinate with the internal "noise-police" tool. The tool will minimize noise on devbig by assigning all cgroups except "workflow.slice" to NUMA node 1, and assign the cgroup "workflow.slice" with dedicated NUMA node 0. The "systemd-run" command is to run TorchBench under "workflow.slice" cgroup.